recipes/cpython/all/conandata.yml

```
@@ -50,6 +50,8 @@ patches:
       base_path: "source_subfolder"
     - patch_file: "patches/3.8.12-0004-disable-macos-tcltk.patch"
       base_path: "source_subfolder"
+    - patch_file: "patches/3.8.12-0005-_ctypes-proj.patch"
+      base_path: "source_subfolder"
```

```suggestion
      base_path: "source_subfolder"
```
Could you please add extra information, like `patch_description` and `patch_type`. Also, if it's part of upstream, please, point it on `patch_source`.

More information: https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches-fields